### PR TITLE
EWL-855-hero-live-event-page

### PR DESCRIPTION
**Pull request by narajaon**
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: src/modules/LiveEventHero/index.stories.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { LiveEventHero } from '../..';
+
+const LiveEventHeroStories = storiesOf('Modules|LiveEventHero', module);
+LiveEventHeroStories.add('Live Event Hero', () => <LiveEventHero title="ROLAND GARROS" subtitle="3RD ROUND" />);
```
Comment (davidgarry) [inline src/modules/LiveEventHero/index.stories.js]: Can you add knobs to be able to change the title and the subtitle in the storybook ?

File: src/modules/LiveEventHero/index.js [hunk 2]
```diff
@@ -1,258 +1,23 @@
...
-  `)};
-`;
-
-export const StyledMatchInfo = styled(StyledCourtInfo)`
-  color: ${colors.coreLightMinus1};
-  margin-bottom: 24px;
-  ${breakpoints.medium(css`
-    margin-bottom: 24px;
-  `)};
+  color: ${coreLightMinus1};
```
Comment (JulienKode) [inline src/modules/LiveEventHero/index.js]: May be we can start using `colors` and `typo` from `theme` https://github.com/EurosportDigital/web-toolkit/blob/master/src/theme.js

What did you think ?

File: src/modules/LiveEventHero/index.stories.js (hunk 1 above)
Comment (narajaon) [reply src/modules/LiveEventHero/index.stories.js]: Good catch !

File: src/modules/LiveEventHero/index.js (hunk 2 above)
Comment (narajaon) [reply src/modules/LiveEventHero/index.js]: Great idea, thanks

File: src/modules/LiveEventHero/index.stories.js [hunk 3]
```diff
@@ -1,9 +1,13 @@
...
 import { storiesOf } from '@storybook/react';
 import { text } from '@storybook/addon-knobs';
+import { ThemeProvider } from 'emotion-theming';
+import theme from '../../theme';
 import { LiveEventHero } from '../..';
 
 const LiveEventHeroStories = storiesOf('Modules|LiveEventHero', module);
 LiveEventHeroStories.add('Live Event Hero', () => (
-  <LiveEventHero title={text('title', 'ROLAND GARROS')} subtitle={text('subtitle', '3RD ROUND')} />
+  <ThemeProvider theme={theme}>
```
[the remark this answers is not in the document]
Comment (JulienKode) [reply src/modules/LiveEventHero/index.stories.js]: https://github.com/EurosportDigital/web-toolkit/blob/master/.storybook/config.js#L24

**Pull request merged**